Question 2.js

```diff
@@ -0,0 +1,19 @@
+function fibonacciEvenSum(total){
+    let sum = 0;
+    let num1 = 1;
+    let num2 = 2;
+    let temp = 0;
+
+    while(num1 + num2 <= total){
+        temp = num1 + num2;
+        num1 = num2;
+        num2 = temp;

```

Comment:
Did you skip adding the first even term - `2`? I think this algorithm doesn't account for it.